### PR TITLE
docs(build-cop): add info about Bintray conflicts

### PR DESCRIPTION
**Pull request by plumpy**

### CODE REVIEW

**Review by maggieneterval — APPROVED** [review]
📥 ⚔️ ➡️ ☮️ ✅

File: community/contributing/nightly-builds/index.md [hunk 1]
```diff
@@ -150,3 +149,28 @@ All processes are run as the `jenkins` user and most of the useful links are in
 ```bash
 $ sudo su - jenkins
 ```
+
+### Common Build Failures
```
Comment (dorbin) [inline community/contributing/nightly-builds/index.md]: There's already an h3-level `Build failures` heading above Should this go under there?

File: community/contributing/nightly-builds/index.md [hunk 2]
```diff
@@ -150,3 +149,28 @@ All processes are run as the `jenkins` user and most of the useful links are in
...
+
+If an artifact is uploaded to the Bintray repository but never published
+(either because of a transient Bintray error or an interrupted build), you'll
+get an error like this:
+
+> Bintray API Request 'create version 0.20.0-20200512192702' failed with HTTP response 409 Conflict
+
+Delete the artifact to resolve the issue.
+
+First, navigate to the specific version [in the Bintray
```
Comment (dorbin) [inline community/contributing/nightly-builds/index.md]: Can you convert these last paragraphs into steps?

**Review by dorbin — COMMENTED** [review]
Small matter of style, all these headings should be sentence case. 
( s/Bintry Conflicts/Bintray conflicts/ )

File: community/contributing/nightly-builds/index.md (hunk 1 above)
Comment (plumpy) [reply community/contributing/nightly-builds/index.md]: Yep, thanks.

File: community/contributing/nightly-builds/index.md (hunk 2 above)
Comment (plumpy) [reply community/contributing/nightly-builds/index.md]: Done, much better, thanks!

File: community/contributing/nightly-builds/index.md [hunk 3]
```diff
@@ -114,6 +114,33 @@ on what specificially went wrong.
...
+
+##### Bintray Conflicts
+
+If an artifact is uploaded to the Bintray repository but never published
+(either because of a transient Bintray error or an interrupted build), you'll
+get an error like this:
+
+> Bintray API Request 'create version 0.20.0-20200512192702' failed with HTTP response 409 Conflict
+
+Follow these steps to delete the artifact and resolve the issue.
```
Comment (dorbin) [inline community/contributing/nightly-builds/index.md]:
```suggestion
Follow these steps to delete the artifact and resolve the issue:
```

File: community/contributing/nightly-builds/index.md [hunk 4]
```diff
@@ -114,6 +114,33 @@ on what specificially went wrong.
...
+
+1. Navigate to the specific version [in the Bintray
+   repository](https://bintray.com/beta/#/spinnaker-releases/jars?tab=packages)
+
+1. Click on the Spinnaker repository that had the failure. (If you don't see
+it, click to the next page; there are only 10 items per page for some reason.)
```
Comment (dorbin) [inline community/contributing/nightly-builds/index.md:133]: 😃

**Review by dorbin — APPROVED** [review]
One small suggestion, but lgtm.
🥇

**Pull request merged**